deptofdefense/move.mil
# Customer service

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…


## Summary of Changes

Adds the Customer Service page content, loading the branch-specific customer service numbers from the database.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. `git checkout customer-service`,
1. set up development dependencies according to [CONTRIBUTING.md](https://github.com/deptofdefense/move.mil/blob/master/CONTRIBUTING.md),
1. `bundle install`,
1. `bin/rails db:migrate`,
1. `bin/rails db:seed`,
1. `bin/rails server`, and
1. load up http://localhost:3000/customer-service in your Web browser of choice.

## Screenshots

### Desktop
![fireshot capture 43 - customer service move mil - http___localhost_3000_customer-service](https://user-images.githubusercontent.com/26554826/29045950-63eca36e-7b93-11e7-9b4b-3892003fd315.png)

### Mobile
![fireshot capture 47 - customer service move mil - http___localhost_3000_customer-service](https://user-images.githubusercontent.com/26554826/29045949-63e84f6c-7b93-11e7-84e7-19205e72f0ed.png)
